docker_xylem/service.py

```diff
@@ -28,8 +28,14 @@ def __init__(self, config):
 
         self.xylem_host = config['host']
         self.xylem_port = config.get('port', 7701)
-        self.mount_path = config.get('mount_path', '/var/lib/docker/volumes')
-
+        self.mount_path = config.get(
+            'mount_path',
+            '/var/lib/docker-xylem/volumes'
+        )
+        self.old_paths = {
+            '/var/lib/docker/volumes',
+            'var/lib/docker-xylem',
```

Comment:
I don't think we need `var/lib/docker-xylem`. Also, `old_paths` should probably be configurable and not default to having `/var/lib/docker/volumes` in it.